quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/GizmoMemberAccessorEntityEnhancer.java

```diff
@@ -417,6 +419,76 @@ public String generateGizmoBeanFactory(ClassOutput classOutput, Set<Class<?>> be
         return generatedClassName;
     }
 
+    private static class QuarkusGizmoSolutionClonerImplementor extends GizmoSolutionClonerImplementor {
```

Comment:
The class is IMO a big too big to be an inner class. Consider making it a top-level class; it's already static, so you don't need the reference to the outer class.